tests/Unit/Helpers/Evolution/DiscontinuousGalerkin/BoundaryConditions.hpp

```diff
@@ -0,0 +1,614 @@
+    tmpl::list<BoundaryConditionVolumeTags...> /*meta*/, const double epsilon) {
+  CAPTURE(FaceDim);
+  CAPTURE(python_module);
+  CAPTURE(face_points);
+  CAPTURE(use_moving_mesh);
+  CAPTURE(epsilon);
+  CAPTURE(pretty_type::short_name<BoundaryCorrection>());
+  const size_t number_of_points_on_face = face_points.product();
+
+  using variables_tag = typename System::variables_tag;
+  using variables_tags = typename variables_tag::tags_list;
+  using flux_variables = typename System::flux_variables;
+  using dt_variables_tags = db::wrap_tags_in<::Tags::dt, variables_tags>;
+
+  constexpr bool uses_ghost =
+      BoundaryCondition::bc_type ==
+          ::evolution::BoundaryConditions::Type::Ghost or
+      BoundaryCondition::bc_type ==
+          ::evolution::BoundaryConditions::Type::GhostAndTimeDerivative;
+  constexpr bool uses_time_derivative_condition =
+      BoundaryCondition::bc_type ==
+          ::evolution::BoundaryConditions::Type::TimeDerivative or
+      BoundaryCondition::bc_type ==
+          ::evolution::BoundaryConditions::Type::GhostAndTimeDerivative;
+
+  // List that holds the inverse spatial metric if it's needed
+  using inverse_spatial_metric_list =
+      ::evolution::dg::Actions::detail::inverse_spatial_metric_tag<System>;
+  constexpr bool has_inv_spatial_metric =
+      ::evolution::dg::Actions::detail::has_inverse_spatial_metric_tag_v<
+          System>;
+
+  // Set up tags for boundary conditions
+  using bcondition_interior_temp_tags =
+      typename BoundaryCondition::dg_interior_temporary_tags;
+  using bcondition_interior_prim_tags =
+      typename ::evolution::dg::Actions::detail::get_primitive_vars<
+          System::has_primitive_and_conservative_vars>::
+          template boundary_condition_interior_tags<BoundaryCondition>;
+  using bcondition_interior_evolved_vars_tags =
+      typename BoundaryCondition::dg_interior_evolved_variables_tags;
+  using bcondition_interior_dt_evolved_vars_tags =
+      ::evolution::dg::Actions::detail::get_dt_vars_from_boundary_condition<
+          BoundaryCondition>;
+  using bcondition_interior_deriv_evolved_vars_tags =
+      ::evolution::dg::Actions::detail::get_deriv_vars_from_boundary_condition<
+          BoundaryCondition>;
+  using bcondition_interior_tags = tmpl::remove_duplicates<tmpl::append<
+      tmpl::conditional_t<has_inv_spatial_metric,
+                          tmpl::list<::evolution::dg::Actions::detail::
+                                         NormalVector<FaceDim + 1>>,
+                          tmpl::list<>>,
+      bcondition_interior_evolved_vars_tags, bcondition_interior_prim_tags,
+      bcondition_interior_temp_tags, bcondition_interior_dt_evolved_vars_tags,
+      bcondition_interior_deriv_evolved_vars_tags>>;
+
+  std::uniform_real_distribution<> dist(-1., 1.);
+
+  // Compared to the evolution code itself, in the test we only need the tags
+  // that are passed to the boundary condition.
```

Comment:
I'll remove, this is trying to say "Instead of checking that the output of the boundary condition _and_ the boundary correction are correct, we only check that what the boundary condition returns is correct, leaving verification of the boundary correction to a separate test."

Reply:
Thanks for clarifying, but yeah removing sounds appropriate 👍